### PR TITLE
Format review: some typos and grammar errors fixes

### PR DESCRIPTION
**Pull request by mere-teresa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,29 +13,29 @@
   <img width="300" height="150" src="./assets/delete.gif" alt="Areas delete">
 </p>
 
-Areas aims to provide a flexible and customizable environment to create custom UI layouts using [Vue.js](https://vuejs.org/).
+Areas aim to provide a flexible and customizable environment to create custom UI layouts using [Vue.js](https://vuejs.org/).
```
Comment (Kurtil) [inline README.md:16]: Areas is singular, this is the name of the component, not some areas.

File: README.md [hunk 2]
```diff
@@ -85,58 +85,59 @@ const areasCfg = {
...
+Result:
 
 <img width="400" src="./assets/areas-simple-example.png" alt="Areas simple example">
 
 # API
 
 ## Configuration
 
-Areas needs an object as cfg props :
+Areas need an object as cfg props:
```
Comment (Kurtil) [inline README.md:96]: Areas is singular, this is the name of the component, not some areas.

File: README.md (hunk 1 above)
Comment (mere-teresa) [reply README.md:16]: Sorry for my mistake.

File: README.md [hunk 3]
```diff
@@ -214,34 +215,34 @@ The `$area` object will provide the following properties :
...
 | contentComponent | The Vue.js component instance of the first child of the area. (It is different from area.$children[0] due to implementation) |
 | areas | The Areas Vue.js component instance. |
-| onChange(handler) | A method that accepts a callback (handler) as parameter. The handler will be called when area change (swap) with newAreaId and OldAreaId as parameters. |
+| onChange(handler) | A method that accepts a callback (handler) as parameter. The handler is called when area change (swap) with newAreaId and OldAreaId as parameters. |
 | offChange(handler) | A method that accepts a callback (handler) as parameter to stop listening to area change with this handler. |
 
 # Development
 
-Build on change for development and serve :
+Build on-change for development and server:
```
Comment (Kurtil) [inline README.md:223]: keep serve

File: README.md [hunk 4]
```diff
@@ -145,13 +146,13 @@ Methods on Areas instance :
 | `changeAreaContent(areaId, contentCfg)` | `areaId`: **Required**. The id of the area to change the content. `contentCfg`: **Required**. An [`area content`](#area) object.  | Change the content of an area. |
 | `setMode(mode)` | `mode`: **Required**. See [Modes](#Modes) | Change Areas mode. |
 | `deleteArea(areaId)` | `areaId`: **Required**. | Delete area with this id. |
-| `splitArea(areaId, way, percentage = 50, insertNewAfter = true)` | `areaId`: **Required**. `way`: **Required**. "vertical" or "horizontal". `percentage`, the percentage from left or top. `insertNewAfter`, if the new area will be inserted after or before the splitted one. | Split area. |
+| `splitArea(areaId, way, percentage = 50, insertNewAfter = true)` | `areaId`: **Required**. `way`: **Required**. "vertical" or "horizontal". `percentage`, the percentage from left or top. `insertNewAfter`, if new the area is inserted after or before the splitted one. | Split area. |
```
Comment (Kurtil) [inline README.md:149]: "if the new"

File: README.md [hunk 5]
```diff
@@ -85,58 +85,59 @@ const areasCfg = {
...
 
-### `area` :
+### `area`:
 | Property | Type | Description |
 | :--- | :--- | :--- |
-| `componentIndex` | `number` | **Required**. A valid index of the `cfg.components` array, or null for the default component. |
-| `name` | `string` | **Optional**. A name that can be used to retrieve area content. |
-| `cfg` | `object` | **Optional**. A [data object](https://vuejs.org/v2/guide/render-function.html#The-Data-Object-In-Depth) used to pass props, event listener... to the component instance. |
+| `componentIndex` | `number` | **Required**. A valid index of the `cfg.components` array, or `null` for the default component. |
+| `name` | `string` | **Optional**. A name, useful to retrieve area content. |
```
Comment (Kurtil) [inline README.md:112]: used

**Pull request merged**